File: Ruby/01 - Introduction to Programming/Task10/control.rb
#Task10 - control.rb
#------------------------------------------------------------#
#request input from user
print "What is your age? "
age = gets.chomp().to_i

#print message according to age
if age>=18
	print("You are old enough!")
elsif age>16
	print("Almost there")
else
	print("You're just too young!")
end

File: Java/01 - Introduction to Programming/Task10/control.java
import java.util.*;

public class control
{
    public static void main(String[] args)
    {
        Scanner keyboardInput = new Scanner(System.in);
        
        //request input from user
        System.out.print("What is your age? ");
        int age = Integer.parseInt(keyboardInput.nextLine());

        //print message according to age
        if (age>=18)
        {
            System.out.print("You are old enough!");
        }
        else if (age>16)
        {
            System.out.print("Almost there");
        }
        else 
        {
            System.out.print("You're just too young!");
        }

        keyboardInput.close();
    }
}